### DISCUSSION TITLE
pip3 install kornia error

### DISCUSSION THREAD

**Discussion by aresoo** [kornia/kornia#3053] [Potential Issue]
Whoever knows how to deal with this issue, I would be extremely grateful.

Collecting kornia
  Using cached kornia-0.7.3-py2.py3-none-any.whl.metadata (7.7 kB)
Collecting kornia-rs>=0.1.0 (from kornia)
  Using cached kornia_rs-0.1.7.tar.gz (71 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... error
  error: subprocess-exited-with-error
  
  × Preparing metadata (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [10 lines of output]
      error: failed to parse manifest at `/private/var/folders/pw/qw_7m_g96sx09mgyj682kq_w0000gn/T/pip-install-px7hlw61/kornia-rs_f62afe9786804700a8884c37b5706ac2/kornia-py/Carg
      
      Caused by:
        can't find library `kornia_rs`, rename file to `src/lib.rs` or specify lib.path
      💥 maturin failed
        Caused by: Cargo metadata failed. Does your crate compile with `cargo build`?
        Caused by: `cargo metadata` exited with an error:
      Error running maturin: Command '['maturin', 'pep517', 'write-dist-info', '--metadata-directory', '/private/var/folders/pw/qw_7m_g96sx09mgyj682kq_w0000gn/T/pip-modern-metad
      Checking for Rust toolchain....
      Running `maturin pep517 write-dist-info --metadata-directory /private/var/folders/pw/qw_7m_g96sx09mgyj682kq_w0000gn/T/pip-modern-metadata-fcq86wym --interpreter /Users/byt
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.

**Comment by edgarriba** [comment]
Hi, what kornia version and more importantly python and os ?
